packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -217,8 +211,8 @@
     <Import Project="..\packages\Microsoft.UI.Xaml.2.3.191129002\build\native\Microsoft.UI.Xaml.targets" Condition="Exists('..\packages\Microsoft.UI.Xaml.2.3.191129002\build\native\Microsoft.UI.Xaml.targets')" />
     <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.targets" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.targets')" />
   </ImportGroup>
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.props" />
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.Cpp.targets" />
+  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\external\Microsoft.ReactNative.Uwp.CppApp.props"/>
```

Comment:
>props [](start = 101, length = 5)

Could we put the props file import before source files as it is usually done?

Reply:
I followed the template we created that puts the props next to the targets. 
I think this is another one of those sneaky differences between C# and C++ Project Systems. 
The managed msbuild convention (as well as for msbuild files in nuget packages) is indeed that the props of a nuget package go on top and the targets at the bottom.... BUT it seems that the Cpp team made a different decision and put both at the bottom of the msbuild file, if you see our vcxproj files that can't yet use PackageReference because WinRT is not yet supported on the new project system...

@vmoroz / @jonthysell: Let me know what you prefer. Happy to move the managed .props to the top of this and then our template too.

Reply:
I did, I separated the pattern for csharp and cpp to follow their conventions.

Reply:
I had to import the bundle.props file in the targets. That props file is setup as being imported after the settings..